api/transfer.py

```diff
@@ -234,7 +236,12 @@ def init_file_import(task: UploadTask, project_id: str, is_global_update: bool)
             import_preperator.prepare_label_studio_import(project_id, task)
         else:
             transfer_manager.import_records_from_file(project_id, task)
-        calculate_missing_attributes(project_id, task.user_id)
+        daemon.run(
+            __recalculate_missing_attributes_and_embeddings,
+            project_id,
+            task.user_id,
```

Comment:
not sure if necessary but task.user_id should be a uuid so maybe a str cast is necessary since the recalculate expects a string.

Reply:
Added str()